src/openforms/registrations/contrib/zgw_apis/plugin.py

```diff
@@ -401,6 +438,31 @@ def register_submission(self, submission: Submission, options: dict) -> dict | N
         if submission.has_registrator:
             result["medewerker_rol"] = medewerker_rol
 
+        # Register submission to Objects API if configured
+        if (
+            options.get("objecttype")
+            and options.get("objecttype_version")
+            and options.get("content_json")
+        ):
+            result["objects_api"] = self.register_submission_to_objects_api(
```

Comment:
```suggestion
            result["objects_api_object"] = self.register_submission_to_objects_api(
```
I'd use the same nomenclature as the Objects API itself so that looking at the data in isolation makes a bit more sense :)